### ISSUE TITLE
Can't fetch translations from server with on ssr

### ISSUE THREAD

**Issue by Ceditvodu** [i18next/i18next-http-backend#70]
## 🐛 Bug Report

If I use `useSuspense: false` and `wait: true`. It seems like fetching only default translation file, but target one in result looks like empty object: `{}` in a console log.
In case when I use `useSuspense: true` and `wait: false`. It shows for a second next error: 
"ReactDOMServer does not yet support Suspense."
And then it renders the page but without component that uses translation... 
But the translation will be presented in console log.

## To Reproduce

Test translation files serves with [http-server](https://www.npmjs.com/package/http-server)

### next-i18next.config.js
```js
const I18nextChainedBackend = require('i18next-chained-backend/dist/cjs/i18nextChainedBackend');
const I18NextHttpBackend = require('i18next-http-backend/cjs');

module.exports = {
  debug: true,
  lng: 'ru',
  i18n: {
    defaultLocale: 'ru',
    locales: ['ru', 'ua', 'pl'],
  },
  react: {
    useSuspense: false,
    wait: true
  },
  fallbaclLng: 'ua',
  localePath: "./public/static/locales",
  serializeConfig: false,
  use: [I18nextChainedBackend],
  partialBundledLanguages: true,
  backend: {
    backends: [
      I18NextHttpBackend,
    ],
    backendOptions: [
      {
        loadPath: 'http://localhost:8086/{{lng}}/{{ns}}.json',
        crossDomain: true,
        requestOptions: {
          mode: 'no-cors',
          cache: 'default',
        },
      },
    ],
  },
};
```

### next.config.js
```js
const { i18n } = require('./next-i18next.config')

module.exports = {
  i18n,
};
```

### _app.tsx
```tsx
import 'styles/globals.css';
import type { AppProps, AppContext } from 'next/app';
import { appWithTranslation } from 'next-i18next'
import { reduxWrapper } from 'store';
import Head from 'next/head';
import nextI18NextConfig from '../next-i18next.config';

function MyApp({ Component, pageProps }: AppProps | any) {
  return (
    <>
      <Head>
        <title>some title</title>
      </Head>
        <Component {...pageProps} />
    </>
  );
}

MyApp.getInitialProps = async ({ Component, ctx }: AppContext) => {
  const pageProps = Component.getInitialProps
    ? await Component.getInitialProps(ctx)
    : {};
  return { pageProps: pageProps };
};

export default reduxWrapper.withRedux(appWithTranslation(MyApp, nextI18NextConfig));
```

### page.tsx
```tsx
import Head from 'next/head';
import React from 'react';
import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
import { GetServerSideProps } from 'next';
import nextI18NextConfig from '../next-i18next.config.js';
import TargetComponent from '@components/TargetComponent';

const Page = (props) => {
  return (
    <>
      <Head>
        <title>page</title>
      </Head>
      <TargetComponent>content</TargetComponent>
    </>
  );
};

export const getServerSideProps: GetServerSideProps = async ({
  locale,
}: any) => {
  return {
    props: {
      ...(await serverSideTranslations(locale, ['header'], nextI18NextConfig)),
    },
  };
};

export default Page;
```

### TargetComponent.tsx
```tsx
import { useTranslation } from 'next-i18next';
import React from 'react';

export const TargetComponent = function (props) {
  const [t, i18n, ready] = useTranslation('example', { 
    useSuspense: false 
  });
  console.log('ready', ready, i18n,  i18n.getResourceBundle(i18n.language, 'header'));
  return (
    <div>
      {ready ? t('warenyky'): 'loading'}
    </div>
  );
};
```

### header.json
```json
{
  "warenyky": "вареники"
}
```

### log when useSuspens: false, wait: true
<img width="964" alt="image" src="https://user-images.githubusercontent.com/8017544/121805769-45301e80-cc55-11eb-876c-b1fb9e945ce2.png">

### log when useSuspens: true, wait: false
<img width="904" alt="image" src="https://user-images.githubusercontent.com/8017544/121805926-1f574980-cc56-11eb-9d65-3ea7d824a665.png">
<img width="520" alt="image" src="https://user-images.githubusercontent.com/8017544/121805941-3433dd00-cc56-11eb-866a-3802413275de.png">
<img width="1128" alt="image" src="https://user-images.githubusercontent.com/8017544/121805957-4c0b6100-cc56-11eb-82af-d6e94a0ece3c.png">

## Expected behavior

The possibility to use backend stored translation with `next-i18next`, `i18next-http-backend` and `i18next-chained-backend`.

## Environment

- *runtime version*:  node v14, safari
- *next-i18next version*: 7.0.1
- *i18next-http-backend version*: 1.2.6
- *i18next-chained-backend version*: 3.0.2
- *next version*: 10.2.3
- *os*: Mac

**Comment by adrai** [comment]
Some stuff I noticed:

- the `wait` option does not exist anymore... (see log warning in your screenshot)
- why are you using i18next-chained-backend?
- the only namespace that seems not to have loaded, seems to be the common namespace for ru language... (does a common.json exist for the ru language?)
- seems you're using next-i18next, why do you need such a strange setup? why not using a default next-i18next setup? => https://github.com/isaachinman/next-i18next/tree/master/examples/simple

My suggestions:
- remove i18next-chained-backend, and if needed use only i18next-http-backend
- make sure all json files are present
- update your next.js next-i18next version and try to use the normal setup: https://github.com/isaachinman/next-i18next/tree/master/examples/simple

**Comment by Ceditvodu** [comment]
@adrai 

- I see, thanks you for this information
- The i18next-chained-backend I want to use for fallback to local files with i18next-resources-to-backend
- It is look pretty common except several additional parameters... Did you mean some certain? 

- After removing nothing was changed. 
- Yes, all are present
- All the packages are up to date

The only one that I've resolved during experements is that `partialBundledLanguages: true`, somehow making unable to get correct translation... And if I will turn it off, the translation will appear! BUT, the "ReactDOMServer does not yet support Suspense." message will be present on start for 2 seconds. Is there way to avoid it somehow? 

<img width="1021" alt="image" src="https://user-images.githubusercontent.com/8017544/121814369-5559f500-cc79-11eb-9c99-611d2d14bab5.png">
